modules/oe_starter_content_news/oe_starter_content_news.info.yml

```diff
@@ -0,0 +1,22 @@
+name: OpenEuropa Starter Content News
+type: module
+description: OpenEuropa Starter Content News.
+core_version_requirement: ^9.2
+package: OpenEuropa Starter Content
+dependencies:
+  - drupal:datetime
+  - drupal:media
```

Comment:
This dependency should be inferred from the featured media field one, so no need.